lib/beaker-pe/install/pe_utils.rb

```diff
@@ -485,7 +485,7 @@ def simple_monolithic_install(master, agents, opts={})
           end
 
           step "Run puppet to setup mcollective and pxp-agent" do
-            on(master, puppet_agent('-t'), :acceptable_exit_codes => [0,2])
+            on(master, puppet_agent('-t'), :acceptable_exit_codes => [2])
```

Comment:
I don't know if this is a safe change.

If a user of this method is deploying with mixed agents, `deploy_frictionless_to_master` will perform a puppet agent run, and if this is before 2016.5.x (services.conf), then the second puppet agent run should return no changes?

Same for the changes below.